Remotion/Web/Development.WebTesting/BrowserSession/Firefox/FirefoxBrowserSession.cs

```diff
@@ -27,15 +30,38 @@ namespace Remotion.Web.Development.WebTesting.BrowserSession.Firefox
   /// </summary>
   public class FirefoxBrowserSession : BrowserSessionBase<IFirefoxConfiguration>
   {
-    public FirefoxBrowserSession ([NotNull] Coypu.BrowserSession value, [NotNull] IFirefoxConfiguration browserConfiguration, int driverProcessId, bool headless)
+    private readonly ConcurrentQueue<BrowserLogEntry> _logEntries;
+
+    public FirefoxBrowserSession (Coypu.BrowserSession value, IFirefoxConfiguration browserConfiguration, int driverProcessId, bool headless)
         : base(value, browserConfiguration, driverProcessId, headless)
     {
+      _logEntries = new ConcurrentQueue<BrowserLogEntry>();
+
+      var bidi = ((IWebDriver)Driver.Native).AsBiDiAsync().Result;
```

Comment:
https://stackoverflow.com/questions/17284517/is-task-result-the-same-as-getawaiter-getresult

> Task.GetAwaiter().GetResult() is preferred over Task.Wait and Task.Result because it propagates exceptions rather than wrapping them in an AggregateException. 

Given that, I would prefer the more verbose syntax.